packages/editor/src/browser/editor-manager.ts

```diff
@@ -189,6 +189,13 @@ export class EditorManager extends NavigatableWidgetOpenHandler<EditorWidget> {
         };
     }
 
+    /**
+     * Save all editors.
+     */
+    async saveAllEditors(): Promise<void> {
+        const widgets = this.shell.widgets.filter(widget => widget instanceof EditorWidget);
+        Promise.all(widgets.map(Saveable.save));
+    }
```

Comment:
There is no need to filter down the list of `EditorWidget`, it is already available with:

```ts
this.all
```